README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolkit** for defining, building and generating **APIs, documentation and SDKs**. Bring an existing OpenAPI definition or describe your endpoints in a Fern specification. Fern generates backend code, specifications in various formats, and client libraries with a single command.
 
-### 🌿 SDKs
+This repository contains the Fern **compiler**. The compiler transforms an API definition into various code artifacts, such as client libraries, SDKs, or even full-fledged **Express**, **Fast API** or **Spring Boot** backends.
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+In short, Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering business value**.
 
-### 🌿 Server-side code generation
+# Capabilities
 
-We automatically generate lots of boilerplate on the server side, like Pydantic models for FastAPI and Jersey interfaces for Spring Boot. We also add compile-time validation that all your endpoints are being served correctly.
+Currently, the Fern compiler can generate the following types of artifacts:
 
-### 🌿 Postman Collection
+### 🌿 Client Libraries and SDKs
```

Comment:
Want to mention the languages we support? (TS, Python, Java, Go) and suggest that folks +1 a GitHub Issue for ones that are on the roadmap (C#, Ruby, Rust)

Reply:
I think its also worth explicitly acknowledging that we are using two terms here to mean the same thing "Client LIbrary" and "SDK".